examples/text-generation/run_generation.py

```diff
@@ -312,6 +302,35 @@ def setup_parser(parser):
         help="Run multi card with the specified parallel strategy. Choices are 'tp' for Tensor Parallel Strategy or 'none'.",
     )
 
+    parser.add_argument(
+        "--run_partial_dataset",
+        action="store_true",
+        help="Run the inference with dataset for specified --n_iterations(default:5)",
+    )
+
+    quant_parser_group = parser.add_mutually_exclusive_group()
+    quant_parser_group.add_argument(
+        "--load_quantized_model_with_autogptq",
+        action="store_true",
+        help="Load an AutoGPTQ quantized checkpoint using AutoGPTQ.",
+    )
+    quant_parser_group.add_argument(
+        "--disk_offload",
+        action="store_true",
+        help="Whether to enable device map auto. In case no space left on cpu, weights will be offloaded to disk.",
+    )
+    quant_parser_group.add_argument(
+        "--load_quantized_model_with_inc",
+        action="store_true",
+        help="Load a Huggingface quantized checkpoint using INC.",
+    )
+    quant_parser_group.add_argument(
+        "--quantized_inc_model_path",
+        type=str,
+        default=None,
+        help="Path to neural-compressor quantized model, if set, the checkpoint will be loaded.",
+    )
```

Comment:
I don't understand the difference between these two arguments, can you please make it clearer?

Reply:
The first: `load_quantized_model_with_inc`, is loading a checkpoint from Huggingface in GPTQ format, like `TheBloke/Llama-2-7b-Chat-GPTQ`
The second: `quantized_inc_model_path`, is loading a model checkpoint that was converted and quantized using INC.
Like the original Meta Llama2-7b model, and a local path of INC.
You can see an explanation in the readme:
examples/text-generation/README.md

Reply:
Okay I see. Both argument names still sound confusing to me though. Maybe I would rename `quantized_inc_model_path` into `local_quantized_inc_model_path` so that it's clearer this should not be used to load a model from the hub. WDYT?

Reply:
that is ok if it makes clear to user